src/bioregistry/data/contexts.json

```diff
@@ -38,4 +43,4 @@
       "ols"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
The extra new line character after the final closing bracket needs removing. That’s what is causing the linter test to fail.

Reply:
Actually that’s not enough, the keys must also be sorted for the linter to be happy.

Reply:
Updated PR in #906.

Reply:
Thanks!